Sources/AWSLambdaRuntimeCore/LambdaHandler.swift

```diff
@@ -44,45 +46,129 @@ public protocol LambdaHandler: EventLoopLambdaHandler {
     ///
     /// - Returns: A Lambda result ot type `Output`.
     func handle(_ event: Event, context: LambdaContext) async throws -> Output
```

Comment:
I would like to see `Context` and `InitializationContext` added as `typealias`es of lambda handlers, which should further simplify the interface.

Reply:
`Context` is a very generic name. I'm not sure, if I would be a fan of this change.

Reply:
I think they’re specific enough for a `LambdaHandler`👀 Especially when AWS Lambda and other platforms are widely adopting the term “context” and “context object”.

Reply:
I would see this as a potential extension point, where we can introduce custom `Context` types by `ExpessibleByLambdaContext` protocol to customize their own context instead of extending `LambdaContext` (which is bad practice).

Reply:
I think this is a good topic for a separate PR